### PR TITLE
topic1

### PR DESCRIPTION
**Pull request by Rodrigocvs**

### CODE REVIEW

File: topic11/.gitignore.rtf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1561\cocoasubrtf600
```
Comment (sebastianscatularo) [inline topic11/.gitignore.rtf]: this file is not the correct one, your .gitignore should not have .rtf extension

File: topic11/.idea/compiler.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sebastianscatularo) [inline topic11/.idea/compiler.xml]: this folder should be ignored

**Comment by sebastianscatularo** [comment]
you have some files that must be ignored, let us know if you need help with that.

File: topic11/pom.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.3.1</version>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
```
Comment (EduardoUTN) [inline topic11/pom.xml]: It's ok. Check later how you can skip test by properties:
https://www.mkyong.com/maven/how-to-skip-maven-unit-test/

**Pull request closed without merging**